Jenkinsfile

```diff
@@ -45,3 +46,18 @@ node {
     throw e
   }
 }
+
+// Run schema tests outside of 'node' definition, so that they do not block the
+// original executor while the downstream tests are being run
+if (env.BRANCH_NAME != 'master') {
```

Comment:
If I remember correctly, the current CI setup has tests in apps are run regardless of the branch, however, I think the overall job status is only dependent on all of the other builds succeeding when building the master branch.

This looks like the app tests are only run when not building master, which I think is different from how the CI works currently. Is this an intentional change?